### PR TITLE
core(unsized-images): pass with explicit aspect-ratio

### PR DESCRIPTION
**Pull request by adamraine**
Fixes https://github.com/GoogleChrome/lighthouse/issues/12218

### CODE REVIEW

File: lighthouse-core/audits/unsized-images.js [hunk 1]
```diff
@@ -86,13 +86,17 @@ class UnsizedImages extends Audit {
...
     const htmlWidthIsExplicit = UnsizedImages.doesHtmlAttrProvideExplicitSize(attrWidth);
     const cssWidthIsExplicit = UnsizedImages.doesCssPropProvideExplicitSize(cssWidth);
     const htmlHeightIsExplicit = UnsizedImages.doesHtmlAttrProvideExplicitSize(attrHeight);
     const cssHeightIsExplicit = UnsizedImages.doesCssPropProvideExplicitSize(cssHeight);
+    const explicitAspectRatio = UnsizedImages.doesCssPropProvideExplicitSize(cssAspectRatio);
     const explicitWidth = htmlWidthIsExplicit || cssWidthIsExplicit;
     const explicitHeight = htmlHeightIsExplicit || cssHeightIsExplicit;
-    return explicitWidth && explicitHeight;
+    return explicitWidth && explicitHeight ||
+      explicitWidth && explicitAspectRatio ||
```
Comment (connorjclark) [inline lighthouse-core/audits/unsized-images.js:98]: I'm sure precedence rules make this work as you intended, but whether AND or OR comes first is not very intuitive, so for readability I'd suggest parentheses even when the grammar is not actually ambiguous.

alternatively: 
`explicitWidth && explicitHeight ||
 ((explicitWidth || explicitHeight) && explicitAspectRatio)`

**Review by patrickhulce — COMMENTED** [review]
impl LGTM 

a smoketest would be great here too since all of this working correctly hinges on us getting the protocol styles response right ;) byte efficiency tester should have plenty of examples to modify or copy

File: lighthouse-cli/test/fixtures/perf/unsized-images.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<html>
```
Comment (patrickhulce) [inline lighthouse-cli/test/fixtures/perf/unsized-images.html:1]: license? we're pretty inconsistent about it non-JS files, so you might need to hunt a smidge before finding one

**Review by patrickhulce — APPROVED** [review]
LGTM!

**Pull request merged**